docs/blog/2018-06-14-announcing-gatsby-v2-beta-launch/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Gatsby v2 beta launch!
+date: "2018-06-14"
+author: "Mike Allanson"
+tags: ["v2", "gatsby"]
+---
+
+We’re excited to announce that today we shipped the first beta for Gatsby v2! 35 contributors have made over 300 commits to v2 since Gatsby v1 was released in July 2017.
+
+## What's coming in v2?
+
+Gatsby v2 builds on the foundations of v1 to introduce a range of improvements:
+
+- Out With Layouts, In With StaticQuery
+
+  Layouts were a common source of confusion in v1. After an RFC and [informative community discussion](https://github.com/gatsbyjs/rfcs/blob/master/text/0002-remove-special-layout-components.md), we decided to remove them and create `StaticQuery` which allows GraphQL queries to be added to any component. For more information on migrating your layouts, check out our Life After Layouts post (coming soon).
+
+- More consistent APIs
+
+  We’ve renamed `sizes` and `resolutions` to `fluid` and `fixed`, `boundActionCreators` to `actions`, as well as other changes intended to make API names more consistent and prevent common gotchas.
+
+  To see the full list of renamings, take a look at [image queries](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#rename-responsive-image-queries), [actions](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#rename-code-classlanguage-textboundactioncreatorscode-to-code-classlanguage-textactionscode) and the [babel](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#change-code-classlanguage-textmodifybabelrccode-to-code-classlanguage-textoncreatebabelconfigcode) and [webpack](https://next.gatsbyjs.com/docs/migrating-from-v1-to-v2/#change-code-classlanguage-textmodifywebpackconfigcode-to-code-classlanguage-textoncreatewebpackconfigcode) API hooks.
+
+- Hotter Hot Reloading
+
+  Previously, hot reload had been dependent on new webpack builds. On larger sites, that could take ten or fifteen seconds -- more like lukewarm reloading, really. With [_ludicrous mode_](https://twitter.com/gatsbyjs/status/974507205121617920) we’ve decoupled webpack so that hot reloading is always instantaneous.
+
+- Upgraded Dependencies
+
+  These are the some of the least obvious but most important changes. Our dependency upgrades help us take advantage of hundreds of other engineers’ work across the incredible modern JS ecosystem.
```

Comment:
I find "least obvious" a bit odd of phrasing — they are fairly obvious to many people. I'd introduce this by saying something like "Gatsby is built on top of amazing JavaScript libraries. We're delighted to ship the latest improvements from webpack, Babel, and React to you" — or something like that.